scripts/settings.js

```diff
@@ -76,6 +77,24 @@ export function registerSettings() {
         },
     });
 
+    game.settings.register(MODULE, 'customScale', {
+        name: 'Custom Scale',
+        hint: `Customize the escalation increment variable. The formula is: scale = originalScale + customScale * elevation `,
+        scope: 'world',
+        config: true,
+        type: Number,
+        range: {
+            min: 0.001,
+            max: 0.02,
+            step: 0.005
```

Comment:
I would like your feedback about this please
I tested and seems ok, but I would like to hear your input

Reply:
please disregard this, I have changed before commiting and messed up. this is ridiculous wrong

Reply:
I am adding another commit fixing this part
as well adding compatibility with v9